apps-rendering/src/components/embedWrapper.stories.tsx

```diff
@@ -11,7 +11,7 @@ import { EmbedComponentWrapper } from './embedWrapper';
 // popped up after I removed the FC type which was causing problems when I tried to set parameters
 // on an individual story. Ideally, we'd use the type from Storybook directly
 // but https://github.com/storybookjs/storybook/issues/13486
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type -- because 👆
```

Comment:
Tried specifically adding the warning here instead of disabling all of the rules but that just ends up triggering the max line length rule.